examples/local-sqlite.flow.yaml

```diff
@@ -2,14 +2,3 @@
 
 import:
   - all-collections.flow.yaml
-
-journalRules:
-  00_fragment_store:
-    template:
-      fragment:
-        compression_codec: GZIP
-        stores:
-          # Default is to persist to the local file system as a stand-in for cloud storage.
-          - file:///
-        # Max time to wait before writing a fragment file
-        flush_interval: 5m
```

Comment:
> examples: drop journalRules, in favor of new flowctl develop default

Is this change to the defaults made elsewhere? Or are you saying that the existing defaults should be used rather than these overrides?

In either case, should we remove these `journalRules` from the demo repos?

Reply:
The (new in this PR) existing defaults can be used rather than this override. This is in part a test / confirmation that the added defaults work as expected.

I don't think it's necessary to actively remove these from examples. The reason for added default behavior is so that a user can run the `flowctl discover` workflow and get a well-behaved catalog that's ready to run with `flowctl develop` without needing to first add explicit `journalRules`.